ocaml/Makefile

```diff
@@ -1011,6 +1011,11 @@ $(RUNTIME_DIR)/%.npic.$(O): OC_CFLAGS += $(OC_NATIVE_CFLAGS) $(SHAREDLIB_CFLAGS)
 $(RUNTIME_DIR)/%.npic.$(O): OC_CPPFLAGS += $(OC_NATIVE_CPPFLAGS)
 $(DEPDIR)/$(RUNTIME_DIR)/%.npic.$(D): OC_CPPFLAGS += $(OC_NATIVE_CPPFLAGS)
 
+# The major GC performs better with this flag on Intel processors
+# This is a workaround for an Intel CPU bug:
+# https://www.intel.co.uk/content/www/uk/en/support/articles/000055650/processors.html
+$(RUNTIME_DIR)/major_gc.%: OC_CFLAGS += $(INTEL_JCC_BUG_CFLAGS)
+
```

Comment:
Please check: I'm not sure if `$(DEPDIR)/$(RUNTIME_DIR)/major_gc.%` needs to be involved? If they only care about `CPPFLAGS`, then maybe not?

Reply:
I believe this is not needed - I asked the above question because I saw lines such as:
```
$(DEPDIR)/$(RUNTIME_DIR)/%.npic.$(D): OC_CPPFLAGS += $(OC_NATIVE_CPPFLAGS)
```
but `$(D)` seems to be dependency descriptions (and thus need pre-processor flags correctly set).

The flag we intend to add has nothing to do with dependency or pre-processor, only with assembler.